### PR TITLE
Dev1.0

### PR DESCRIPTION
**Pull request by vyshnav73373**
Hi @midhunsuresh202 ,

I created pom.xml please review my code.

Regards,
Vyshnav

### CODE REVIEW

File: mavenpom.xml.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.openjfx</groupId>
+  <artifactId>hellofx</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
```
Comment (ghost) [inline mavenpom.xml.txt:7]: Please change the version 1.0 to 1.2

File: mavenpom.xml.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <javafx.version>21</javafx.version>
+    <javafx.maven.plugin.version>0.0.8</javafx.maven.plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openjfx</groupId>
```
Comment (ghost) [inline mavenpom.xml.txt:19]: change group id from org.openjfx to nexus

**Review by ghost — COMMENTED** [review]
Hi Vyshnav,

I added some comments please work and update pull request.

File: mavenpom.xml.txt (hunk 1 above)
Comment (vyshnav73373) [reply mavenpom.xml.txt:7]: I changed.

File: mavenpom.xml.txt (hunk 2 above)
Comment (vyshnav73373) [reply mavenpom.xml.txt:19]: changed

**Pull request merged**